src/main/events/torrenting/pause-game-seed.ts

```diff
@@ -0,0 +1,20 @@
+import { registerEvent } from "../register-event";
+
+import { DownloadManager } from "@main/services";
+import { dataSource } from "@main/data-source";
+import { Game } from "@main/entity";
+
+const pauseGameSeed = async (
+  _event: Electron.IpcMainInvokeEvent,
+  gameId: number
+) => {
+  await dataSource.transaction(async (transactionalEntityManager) => {
+    await transactionalEntityManager
+      .getRepository(Game)
+      .update({ id: gameId }, { status: "complete", shouldSeed: false });
+  });
```

Comment:
i dont think you need to run this inside a transaction, you are only running one update